### PR TITLE
TOOL-30 Updated to check for gems.locked too.

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -123,26 +123,27 @@ func ensureFastlaneVersionAndCreateCmdSlice(forceVersion, gemfilePth string) ([]
...
-
 	bundleInstallCalled := false
-	if exist, err := pathutil.IsPathExists(gemfileLockPth); err != nil {
-		return nil, "", err
-	} else if !exist {
-		log.Printf("Gemfile.lock not exist at: %s, running 'bundle install' ...", gemfileLockPth)
+	gemfileDir := filepath.Dir(gemfilePth)
+	gemfileLockPth, err := gems.GemFileLockPth(gemfileDir)
+	if err != nil {
+		if os.IsNotExist(err) {
```
Comment (lpusok) [inline main.go]: You used ErrGemLockNotFound, not os error.

File: main.go [hunk 2]
```diff
@@ -123,26 +123,27 @@ func ensureFastlaneVersionAndCreateCmdSlice(forceVersion, gemfilePth string) ([]
...
-	gemfileLockPth := filepath.Join(gemfileDir, "Gemfile.lock")
-
 	bundleInstallCalled := false
-	if exist, err := pathutil.IsPathExists(gemfileLockPth); err != nil {
-		return nil, "", err
-	} else if !exist {
-		log.Printf("Gemfile.lock not exist at: %s, running 'bundle install' ...", gemfileLockPth)
+	gemfileDir := filepath.Dir(gemfilePth)
+	gemfileLockPth, err := gems.GemFileLockPth(gemfileDir)
+	if err != nil {
```
Comment (lpusok) [inline main.go]: PLease update logs to refer to gems lockfile generally, not Gemfile.lock specifically.

File: main.go (hunk 1 above)
Comment (richard-bogdan-bitrise) [reply main.go]: Updated.

File: main.go (hunk 2 above)
Comment (richard-bogdan-bitrise) [reply main.go]: Updated.

File: main.go [hunk 3]
```diff
@@ -140,7 +140,7 @@ func ensureFastlaneVersionAndCreateCmdSlice(forceVersion, gemfilePth string) ([]
 			if exist, err := pathutil.IsPathExists(gemfileLockPth); err != nil {
```
Comment (lpusok) [inline main.go]: This check will fail, even if successful as gemfileLockPth is empty. Will need to call gems.GemFileLockPth again.

Comment (richard-bogdan-bitrise) [reply main.go]: Updated.

**Pull request merged**